services/vrf/mocks/aggregator_v3_interface.go

```diff
@@ -0,0 +1,158 @@
+// Code generated by mockery v2.14.0. DO NOT EDIT.
```

Comment:
If we were building, running, and passing tests without these mocks, why add them back?

Reply:
Removed these with Connors suggestion below